configuration/etl/etl_tables.d/jobs/xdw/jobfact_by_.json

```
@@ -157,8 +157,8 @@
             {
                 "name": "processorbucket_id",
                 "type": "int(4)",
-                "comment": "FACT: Pre-determined processor bucket sizes. References processorbucket.id",
-                "nullable": true
+                "comment": "DIMENSION: Pre-determined processor bucket sizes. References processorbucket.id",
```

```suggestion
                "comment": "DIMENSION: Pre-determined processor bucket sizes. References processor_buckets.id",
```

May as well fix the other typos.